docs/cunumeric/source/user/notebooks/cholesky.ipynb

```diff
@@ -0,0 +1,206 @@
+       "array([[1., 0., 0., ..., 0., 0., 0.],\n",
+       "       [0., 1., 0., ..., 0., 0., 0.],\n",
+       "       [0., 0., 1., ..., 0., 0., 0.],\n",
+       "       ...,\n",
+       "       [0., 0., 0., ..., 1., 0., 0.],\n",
+       "       [0., 0., 0., ..., 0., 1., 0.],\n",
+       "       [0., 0., 0., ..., 0., 0., 1.]])"
+      ]
+     },
+     "execution_count": 2,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "np.linalg.cholesky(np.eye(100))"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "09b58cf2-b7ee-45eb-ade6-4d99d930047b",
+   "metadata": {
+    "tags": []
+   },
+   "source": [
+    "We'd like to get some information about how well Cunumeric's `cholesky` function performs. In order to obtain accurate timings, we need to use the `time` function from `legate.timing`. Let's define a helper function `cholesky_timed` that calls the `time` function for us, and prints out the results as well:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "id": "79d9be8b-eb42-42f0-8162-3d974da3b72a",
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "from legate.timing import time\n",
+    "\n",
+    "def cholesky_timed(A):\n",
+    "    start = time()\n",
+    "    result = np.linalg.cholesky(A)\n",
+    "    stop = time()\n",
+    "    \n",
+    "    n = A.shape[0]\n",
+    "    flops = (n**3) / 3 + 2 * n / 3\n",
+    "    total = (stop - start) / 1000.0\n",
+    "    print(f\"Elapsed Time: {total} ms ({(flops / total):0.2} GOP/s)\")\n",
+    "    \n",
+    "    return result"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "4ebbfeba-c768-47b9-b088-32de999b051f",
+   "metadata": {},
+   "source": [
+    "Now we can define a matrix $A$ to decompose. Let's make some thing a little more interesting than a plain identity matrix:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 4,
+   "id": "0678173d-1eb6-4ad6-b282-9b41e9cb3b40",
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "A = np.eye(1000, dtype=np.float64)\n",
+    "A[:, 2] = 1\n",
+    "A = np.dot(A, A.T)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 5,
+   "id": "06a767f0-b7d7-4eec-8ad8-8d1c127babff",
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Elapsed Time: 43.069 ms (7.7e+06 GOP/s)\n"
+     ]
+    }
+   ],
+   "source": [
+    "L = cholesky_timed(A)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "6507a3d8-9667-4fac-bf50-f36241f4b11b",
+   "metadata": {},
+   "outputs": [],
+   "source": []
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "cdee78c6-d2ab-4823-9023-b6b401b5af32",
+   "metadata": {},
+   "outputs": [],
+   "source": []
+  }
+ ],
+ "metadata": {
+  "kernelspec": {
+   "display_name": "Python 3 (ipykernel)",
+   "language": "python",
```

Comment:
There are two empty cells at the bottom, which looks a little untidy.

Reply:
https://github.com/nv-legate/cunumeric/pull/805/commits/940fe8ad19b139fab627f6db0c80df61d7a90eeb